.drone.star

```diff
@@ -2648,23 +2648,57 @@ def getPipelineNames(pipelines = []):
         names.append(pipeline["name"])
     return names
 
-def calculateTestsToRunBasedOnFilesChanged(ctx):
-    return [
-        {
-            "name": "calculate-diff",
-            "image": OC_CI_NODEJS,
-            "commands": [
-                "bash -x tests/drone/getFilesChanged.sh",
-                "ls -la",
+def skipIfUnchanged(ctx, type):
+
+    if ("full-ci" in ctx.build.title.lower()):
+        return []
+
+    skip_step = {
+        "name": "skip-if-unchanged",
+        "image": "owncloudci/drone-skip-pipeline",
+        "when": {
+            "event": [
+                "pull_request",
             ],
-            "when": {
-                "event": [
-                    "pull_request",
-                ],
-            },
         },
+    }
+
+    base_skip_steps = [
+        "^.github/.*",
+        "^changelog/.*",
+        "^config/.*",
+        "^deployments/.*",
+        "^dev/.*",
+        "^docs/.*",
+        "^packages/web-app-skeleton/.*",
+        "^tests/smoke/.*",
```

Comment:
Do you really want to skip that? Should be at least be removed when Web switches to smoke testing on PRs.